dependencies.props

```diff
@@ -21,15 +21,15 @@
 
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
-    <CoreFxExpectedPrerelease>preview2-25319-04</CoreFxExpectedPrerelease>
-    <CoreClrExpectedPrerelease>preview2-25316-03</CoreClrExpectedPrerelease>
+    <CoreFxExpectedPrerelease>preview1-25320-01</CoreFxExpectedPrerelease>
+    <CoreClrExpectedPrerelease>preview1-25319-05</CoreClrExpectedPrerelease>
```

Comment:
The CoreCLR it's trying to insert is missing https://github.com/dotnet/coreclr/pull/11355/files